sklearn/linear_model/_logistic.py

```diff
@@ -1,5 +1,11 @@
 """
-Logistic Regression
+Logistic Regression(aka logit,MaxEnt) classifier.
+
+For a detailed example of how to use logistic reression,see:
+:ref:`sphx_glr_auto_examples_linear_model_plot_logistic.py`
+
+For more information, refer to the user guide:
+https://scikit-learn.org/regressionstable/modules/linear_model.html#logistic-
```

Comment:
Please revert all of these - they are completely redundant as I mentioned in the previous comment.